campaign/scripts/charsheet/char_spell_detail.lua

```diff
@@ -17,7 +17,6 @@ function update()
 	self["resist"].update(bReadOnly);
 	self["text"].update(bReadOnly);
 	self["duration"].update(bReadOnly);
-	self["spellcost"].update(bReadOnly);
```

Comment:
Is there a particular reason you want to separate the the spell cost out from the cost/maintain field?

Reply:
Oops, this must've been an oversight. I don't want to separate the fields. Will fix in a sec.

Reply:
Hold on, perhaps one of us misunderstood.

The following is my understanding:
In the Ability library, the property for spell cost is "spellcost" with the label "Cost".
In the character sheet, the property for spell cost is "costmaintain" with the label "Cost/Maintain".

My assumption is that these fields are equivalent, just with different names and labels, and that the field deals with both base cost and maintain cost.

Am I understanding correctly?

Reply:
Yes, correct. They are equivalent, just with just with different labels.

Reply:
Then I'm not separating the fields at all, I'm joining them back together. I think a previous commit separated them.

Reply:
Indeed you are. My bad. 👍